### PR TITLE
Add FAQ content to popup

### LINKED ISSUE #224: Add FAQ verbiage to popup
After getting the verbiage from Sammer/operations team, add to the FAQ popup.

Currently no content:
image

### PR DESCRIPTION
**Pull request by ahmedhamodi**
close #224 

Before pic is in the issue ticket.

After:
image

### CODE REVIEW

**Comment by ahmedhamodi** [comment]
Created a tech debt task (#227) to address the large files. All the popups (ie. post a job, FAQ, etc) are loaded in the parent component. Would want to separate it into its own component and render it in. A file should not be 1000 lines long! I will work on this after MVP.

File: src/Components/Dashboard/UserProfile.js [hunk 1]
```diff
@@ -894,7 +894,221 @@ class Landing extends Component {
...
+                </p>
+                <p>
+                  Practice practice practice! Try to simulate the interview
+                  setting and treat the interview like the actual interview by
+                  doing all the preparation work and research before you start.
+                </p>
+                <p>
+                  <b>How many credits does it take to book a Mock Interview?</b>
+                </p>
+                <p>5 credits is required.</p>
```
Comment (w41eed) [inline src/Components/Dashboard/UserProfile.js:1024]: I think these should be "5 credits **are** required" instead of 'is'

**Review by w41eed — APPROVED** [review]
Everything looks good, just maybe change the "5 credits is required" to "5 credits are required" but you can do that later

File: src/Components/Dashboard/UserProfile.js (hunk 1 above)
Comment (ahmedhamodi) [reply src/Components/Dashboard/UserProfile.js]: Good catch!

**Review by w41eed — APPROVED** [review]

**Pull request merged**